src/graphql/triage/response.rs

```diff
@@ -109,7 +109,34 @@ impl super::TriageResponseQuery {
     ) -> Result<Option<TriageResponse>> {
         let store = crate::graphql::get_store(ctx).await?;
         let map = store.triage_response_map();
-        Ok(map.get(&sensor, &time)?.map(Into::into))
+        let response: Option<TriageResponse> = map.get(&sensor, &time)?.map(Into::into);
+
+        if let Some(ref triage_response) = response {
+            let tag_ids: Vec<String> = triage_response
+                .inner
+                .tag_ids()
+                .iter()
+                .map(std::string::ToString::to_string)
+                .collect();
+            info_with_username!(
+                ctx,
+                "Retrieved TriageResponse: id: {}, sensor: \"{}\", time: {}, tag_ids: [{}], remarks: \"{}\"",
+                triage_response.inner.id,
+                sensor,
+                time,
+                tag_ids.join(", "),
+                triage_response.inner.remarks
+            );
```

Comment:
@octoaide, regarding `tag_ids`, I think using `write!` instead of `to_string` + `collect` would perform better. Would you do that?